kotlinlib/src/mill/kotlinlib/android/AndroidHiltSupport.scala

```diff
@@ -68,25 +68,17 @@ trait AndroidHiltSupport extends KspModule with AndroidAppKotlinModule {
 
 }
 
-object AndroidHiltTransform extends ExternalModule with JvmWorkerModule {
+trait AndroidHiltTransform extends ExternalModule with JvmWorkerModule {
 
   override def repositoriesTask: Task[Seq[Repository]] = Task.Anon {
     super.repositoriesTask() :+ AndroidSdkModule.mavenGoogle
   }
 
-  def toolsClasspath: T[Seq[PathRef]] = Task {
-    defaultResolver().classpath(
-      Seq(
-        Dep.millProjectModule("mill-kotlinlib-androidhilt")
-      )
-    )
-  }
```

Comment:
We should keep this task, for multiple reasons:
* User can override to customize, e.g. inject their own patched tooling
* We can cache the classpath in one single location
* We can easily fetch the classpath as part of `prepareOffline` requests

Reply:
ah I thought you wanted it removed, sorry, putting it back